lib/ovirt_metrics.rb

```diff
@@ -43,7 +43,7 @@ def self.connected?
   end
 
   def self.disconnect
-    OvirtHistory.connection.disconnect!
+    OvirtHistory.remove_connection
```

Comment:
@chrisarcand, How would the system behave in case of checking connectivity to the C&U db while metrics collection is running ? Wouldn't such scenario will disconnect all of the connection currently used by the metrics collection process (since disconnect is called at the end of test connectivity) ?